x/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/Jigsaw-Code/outline-sdk/x
 go 1.21
 
 require (
-	github.com/Jigsaw-Code/outline-sdk v0.0.17-0.20240726212635-470a9290ec57
+	github.com/Jigsaw-Code/outline-sdk v0.0.17-0.20240920003233-01a5556214fd
```

Comment:
Let's change this to version 0.0.17 and re-run `go mod tidy`. I believe we can revert the change to the `socks5` package.

Reply:
@fortuna I will get back to exploring and implementing the suggested alternative approach in a bit. That would be a requirement for reverting the changes to `socks5` package (which exposes method to send associate command). I am right now focusing on getting a set of remote measurements for the upcoming event and get back on this PR afterwards.